browser/scss/films/main.scss

```diff
@@ -1,10 +1,32 @@
 /* pdp view */
-.movie-card {
+#product .heading {
+  font-weight: bold;
+  text-transform: uppercase;
+  display: block;
+}
+
+.product-info .text {
+  padding-bottom: 20px;
+  display: block;
+  &.price {
+    font-size: 20px;
+  }
+}
+
+product-image {
     width: 160px;
     height: 240px;
-    float: left;
     margin-left: 20px;
     margin-right: 20px;
+    img {
```

Comment:
If we include CSS changes, can we include pictures next time so that we can get a sense of what the changes look like? Pleez and tankyoo